### ISSUE TITLE
tldr's own entry has a broken example. 

### ISSUE THREAD

**Issue by rbuchberger** [tldr-pages/tldr#3135]
TLDR's own entry has one problematic example:

`tldr -p linux tar`

* The `-p` flag should be replaced with the `-o` flag
* There is no linux entry for `tar`. 

If you run that example, you get this:

```bash

~
❯ tldr -p linux tar
usage: tldr [-h] [-u] [-o {linux,osx,sunos}] [-s SOURCE] [-c] command [command ...]
tldr: error: unrecognized arguments: -p

```
If you then replace -p with -o, you get the following:

```

~
❯ tldr -o linux tar
`tar` documentation is not available. Consider contributing Pull Request to https://github.com/tldr-pages/tldr
```

**Comment by mebeim** [comment]
Please, see my comment on #3136.

**Comment by sbrl** [comment]
Which client are you using, @rbuchberger?

**Comment by rbuchberger** [comment]
The [python client](https://github.com/tldr-pages/tldr-python-client), installed via the [Arch Community repo](https://www.archlinux.org/packages/community/any/tldr/). 

Interestingly, [tealdeer](https://github.com/dbrgn/tealdeer) appears to have this issue as well:

```bash
~
❯ tldr --version
tealdeer v1.1.0

~
❯ tldr -p linux figlet
Unknown flag: '-p'

Usage:

    tldr [options] <command>
    tldr [options]

```

**Comment by waldyrious** [comment]
@rbuchberger this was actually tracked in the Python client's repo, at https://github.com/tldr-pages/tldr-python-client/issues/76. It was fixed recently to `-p`, in https://github.com/tldr-pages/tldr-python-client/pull/77. That ([among other changes](https://github.com/tldr-pages/tldr-python-client/compare/0.4.4...master)) hasn't yet been included in a [release](https://github.com/tldr-pages/tldr-python-client/releases) — the last release was from last year. @felixonmars, is there anything blocking a new release of the package?

> Interestingly, tealdeer appears to have this issue as well

Please open an issue [there](https://github.com/dbrgn/tealdeer/issues) so @dbrgn or others can address it :)

**Comment by sbrl** [comment]
Perhaps we should open an issue against tldr-pages/tldr-python-client about a new release?

**Comment by felixonmars** [comment]
@waldyrious Hi, I have made a 0.5 release of tldr-python-client.
